scripts/calibnet_health_check.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "$1" > preloaded_wallet.key
+
+SNAPSHOT_DIRECTORY="/tmp/snapshots"
+LOG_DIRECTORY="/tmp/log"
+
+FOREST_PATH="forest"
+FOREST_CLI_PATH="forest-cli"
+
+echo "Fetching params"
+$FOREST_CLI_PATH fetch-params --keys
+echo "Downloading snapshot"
+$FOREST_CLI_PATH --chain calibnet snapshot fetch --aria2 -s $SNAPSHOT_DIRECTORY
+
+echo "Importing snapshot and running Forest"
+$FOREST_PATH --chain calibnet --encrypt-keystore false --halt-after-import --height=-200 --import-snapshot $SNAPSHOT_DIRECTORY/*.car
+echo "Checking DB stats"
+$FOREST_CLI_PATH --chain calibnet db stats
+echo "Running forest in detached mode"
+$FOREST_PATH --chain calibnet --encrypt-keystore false --log-dir $LOG_DIRECTORY --detach --save-token ./admin_token
+
+echo "Validating checkpoint tipset hashes"
+$FOREST_CLI_PATH chain validate-tipset-checkpoints
+
+echo "Waiting for sync and check health"
+timeout 30m $FOREST_CLI_PATH --chain calibnet sync wait && $FOREST_CLI_PATH --chain calibnet db stats
+echo "Exporting snapshot"
+$FOREST_CLI_PATH --chain calibnet snapshot export
+
+echo "Verifing snapshot checksum"
+sha256sum -c ./*.sha256sum
+
+echo "Testing js console"
+$FOREST_CLI_PATH attach --exec 'showPeers()'
+
+echo "Validating as mainnet snapshot"
+set +e
+$FOREST_CLI_PATH --chain mainnet snapshot validate $SNAPSHOT_DIRECTORY/*.car --force && \
+{
+    echo "mainnet snapshot validation with calibnet snapshot should fail";
+    $FOREST_CLI_PATH --token "$ADMIN_TOKEN" shutdown --force
+    exit 1;
+}
+set -e
+
+echo "Validating as calibnet snapshot"
+$FOREST_CLI_PATH --chain calibnet snapshot validate $SNAPSHOT_DIRECTORY/*.car --force
+
+echo "Print forest log files"
+ls -hl $LOG_DIRECTORY
+cat $LOG_DIRECTORY/*
+
+echo "Wallet tests"
+
+# The following steps does basic wallet handling tests.
+
+# Amount to send to
+FIL_AMT=500
+# Admin token used when interacting with wallet
+ADMIN_TOKEN=$(cat admin_token)
+# Wallet addresses: 
+# A preloaded address
+ADDR_ONE=t1ac6ndwj6nghqbmtbovvnwcqo577p6ox2pt52q2y
+
+echo "Importing preloaded wallet key"
+$FOREST_CLI_PATH --chain calibnet --token "$ADMIN_TOKEN" wallet import preloaded_wallet.key
+sleep 5s
+
+echo "Exporting key"
+$FOREST_CLI_PATH --chain calibnet --token "$ADMIN_TOKEN" wallet export "$ADDR_ONE" > preloaded_wallet.test.key
+if ! cmp -s preloaded_wallet.key preloaded_wallet.test.key; then
+    echo ".key files should match"
+    $FOREST_CLI_PATH --token "$ADMIN_TOKEN" shutdown --force
+    exit 1
+fi
+
+echo "Fetching metrics"
+wget -O metrics.log http://localhost:6116/metrics
+
+sleep 5s
+
+# Show balances
+echo "Listing wallet balances"
+$FOREST_CLI_PATH --chain calibnet --token "$ADMIN_TOKEN" wallet list
```

Comment:
I wonder if we could have the `FULLNODE_API_INFO="<token goes here>:/ip4/<host>/tcp/<port>/http` env variable to not append the `--token $ADMIN_TOKEN` everywhere?